.gitignore

```
@@ -4,3 +4,4 @@
 
 # Ignore environemnt variables
 .env
+.idea/
```

Thank you!  However also please add coverage, it makes for a smaller PR.

```suggestion
.idea/
coverage
```